docs/implementation.html

```diff
@@ -40,9 +40,9 @@
 
 The goal is to expose all the producer functionality through a single API to the client.
 
-The new producer -
+The Java producer
```

Comment:
I thought about this too. Then I noticed that there are references to "old producer" in the upgrade section. If that doesn't cause a confusion I would also like to use "Kafka producer".